rules/linux/defense_evasion_log_files_deleted.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/11/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies deletion of Linux sensitive system logs. This may indicate an attempt to evade detection or destroy forensic
+evidence on a system.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "System Log Files Deleted"
```

Comment:
Since this is Linux-specific, it may make sense to include that in the name to differentiate it